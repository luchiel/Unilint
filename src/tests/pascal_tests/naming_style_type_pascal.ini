[naming_styles]
class_naming_style = pascal