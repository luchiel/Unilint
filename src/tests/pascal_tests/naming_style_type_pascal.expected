19:3: name camelRecord is not of style pascal
30:3: name underscore_type is not of style pascal
31:3: name CAPS_UNDERSCORE_TYPE is not of style pascal
